### PR TITLE
Update start_here.ipynb

### PR DESCRIPTION
**Pull request by abbycross**
typos and consistency in capitalization/punctuation

### CODE REVIEW

File: qiskit/start_here.ipynb [hunk 1]
```diff
@@ -19,62 +19,62 @@
...
     "\n",
     "\n",
-    "We've collected a core reference set of notebooks in this section outlining the features of Qiskit. We will be keeping them up to date with the latest Qiskit version, currently 0.9. The focus of this section will be how to use Qiskit and not so much on teaching you about quantum computing. For those interested in learning about quantum computing we recommend the awesome learning paths we and the community have put together.\n",
+    "We've collected a core reference set of notebooks in this section outlining the features of Qiskit. We will be keeping them up to date with the latest Qiskit version, currently 0.9. The focus of this section will be how to use Qiskit and not so much on teaching you about quantum computing. For those interested in learning about quantum computing we recommend the awesome [educational material](https://www-dev.quantum-computing.ibm.com/support) we and the community have put together.\n",
     "\n",
```
Comment (jaygambetta) [inline qiskit/start_here.ipynb]: can we not link to the dev. Test the link to the new ibm quantum experience. https://quantum-computing.ibm.com/support

Comment (aasfaw) [reply qiskit/start_here.ipynb]: link works in main website.

**Comment by abbycross** [comment]
Conflicts resolved -- pls review/approve.  Thx!

File: qiskit/start_here.ipynb [hunk 2]
```diff
@@ -38,44 +38,65 @@
...
+    "  * [Mapping circuit qubits to device qubits](terra/device_mapping.ipynb) - How to manually select the mapping of qubits from a circuit to a given device\n",
+    "  * [Reduced backends and noise models](terra/reduced_backends.ipynb) - Working with sub-graphs of a quantum device topology\n",
+    "  * [Monitoring jobs and backends](terra/backend_monitoring_tools.ipynb) - Tools for monitoring jobs and backends\n",
+    "  * [Parallel tools](terra/terra_parallel_tools.ipynb) - Executing tasks in parallel using `parallel_map` and tracking progress\n",
+    "  * [Creating a new provider](terra/creating_a_provider.ipynb) - A guide to integration of a new provider with Qiskit structures and interfaces\n",
+    "  * [Calibrating a qubit](terra/calibrating_a_qubit.ipynb) - An introduction to working with the pulse API by way of calibrating a qubit\n",
+    "  * [Operators overview](terra/operators_overview.ipynb) - Gives a summary of the features and uses of the Operator class\n",
+    "\n",
+    "  \n",
+    "##  3 Qiskit Interacitve Plotting and Jupyter Tools\n",
```
Comment (aasfaw) [inline qiskit/start_here.ipynb]: Interacitve -> Interactive

**Pull request merged**